Gemfile

```diff
@@ -198,3 +198,5 @@ gem 'net-http', '~> 0.3.2'
 gem 'rubyzip', '~> 2.3'
 
 gem 'hcaptcha', '~> 7.1'
+
+gem 'flatware-rspec', '~> 2.1'
```

Comment:
Probably should be up in the `:development` section where the `rspec-chunked` was

Reply:
Good catch - I had `bundle add`'d it during testing and never moved into group. Done.